### PR TITLE
Player movement

### PR DESCRIPTION
**Pull request by ivantjh**
Implement basic movement for player

### CODE REVIEW

**Comment by ivantjh** [comment]
Need to check collision against wall to make it real

File: src/constants.h [hunk 1]
```diff
@@ -53,5 +54,11 @@ const UCHAR DOWN_KEY = VK_DOWN;
 // Sprites
 const char MAP_1_IMAGE[] = "sprites\\level1.png";
 const char MAP_2_IMAGE[] = "sprites\\level2.png";
+const char MALE_PLAYER_IMAGE[] = "sprites\\male-player.png";
```
Comment (chesterhow) [inline src/constants.h]: Would be better to rename this to `PLAYER_MALE_IMAGE[]`

File: src/image.cpp [hunk 2]
```diff
@@ -53,7 +54,7 @@ bool Image::initialize(Graphics *g, int width, int height, int ncols, TextureMan
 		if (cols == 0)
 			cols = 1;                               // if 0 cols use 1
 
-		// configure spriteData.rect to draw currentFrame
+		//// configure spriteData.rect to draw currentFrame
```
Comment (chesterhow) [inline src/image.cpp]: :////

File: src/threechances.cpp [hunk 3]
```diff
@@ -1,10 +1,15 @@
 #include "threechances.h"
-#include <string>
+#include <iostream>
```
Comment (chesterhow) [inline src/threechances.cpp]: Lets standardise and put all `#include`s in header files

File: src/threechances.cpp [hunk 4]
```diff
@@ -20,41 +25,130 @@ ThreeChances::~ThreeChances() {
...
 }
 
 //=============================================================================
 // Update all game items
 //=============================================================================
-void ThreeChances::update() {
-	if (input->wasKeyPressed(RIGHT_KEY)) {
-		if (-map.getX() < map.getWidth() * SCALE - GAME_WIDTH)
-			map.setX(map.getX() - TILE_SIZE);
+void cleanKeys(Input *input, std::map<std::string, bool> *keysPressed) {
```
Comment (chesterhow) [inline src/threechances.cpp]: Move the comment above pls

Comment (chesterhow) [inline src/threechances.cpp]: Can this be more descriptive? e.g. `resetKeysPressedMap`

File: src/threechances.h [hunk 5]
```diff
@@ -10,8 +11,12 @@ class ThreeChances : public Game {
 private:
 	// Game items
 	TextureManager mapTexture;
+	TextureManager malePlayerTexture;
 	Image		   map;
-
+	Image		   malePlayer;
```
Comment (chesterhow) [inline src/threechances.h]: Same here can we switch to `playerMaleTexture` and `playerMale`?

**Review by chesterhow — COMMENTED** [review]
Also for the map can you name the gridded map as `LEVEL1 (w Grid).png` instead of the other way round.

**Pull request merged**